thrift/src/main/java/com/linecorp/armeria/server/thrift/ThriftDocServicePlugin.java

```diff
@@ -168,7 +224,8 @@ static ServiceInfo newServiceInfo(
             });
         }
 
-        return new ServiceInfo(name, functions, classes, endpoints, docStrings.get(name), exampleHeaders);
+        return new ServiceInfo(name, functions, classes, endpoints,
```

Comment:
Yeah, this sounds good to me for example headers + requests. It's similar to what I've done in #423 to avoid having to find all the ```classes``` among the services in the plugins since it's the same code for any plugin.

For docstrings, it's more iffy since there are many different types of ```Info``` like constant, field, etc. We would need to provide utilities to create the keys for any particular type, I guess as static methods on each ```Info``` class - I guess it's not so bad if we do that though.